src/Symfony/Component/HttpFoundation/ParameterBag.php

```diff
@@ -151,6 +176,14 @@ public function getBoolean(string $key, bool $default = false): bool
         return $this->filter($key, $default, \FILTER_VALIDATE_BOOL, ['flags' => \FILTER_REQUIRE_SCALAR]);
     }
 
+    /**
+     * Returns the parameter value converted to boolean or null.
+     */
+    public function getBooleanOrNull(string $key, ?bool $default = null): ?bool
```

Comment:
We use Boolean (long) but Int (short), what about using either `getIntegerOrNull` or `getBoolOrNull`?

Reply:
Yeah, I agree this is weird, but the decision should be taken first on `getBoolean` and `getInt`.
I'll be happy to update my PR as soon as a decision is taken first on which method between `getBoolean` and `getInt` should be deprecated (in favor of `getBool` or `getInteger`)